source/about/self-hosted-subscriptions.rst

```diff
@@ -82,17 +82,27 @@ This process adds additional users to your existing subscription and is not a ne
 
 If you don't have a self-hosted license in the Customer Portal and want to add users, `contact us <https://mattermost.com/contact-us/>`__.
 
-Quarterly report
-^^^^^^^^^^^^^^^^
+Quarterly true-up report
+------------------------
 
-When you subscribe to an annual plan with Mattermost, you agree to provide quarterly reports of the actual number of registered users within your system if you're above the amount you subscribed in your purchase order. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
+When you buy an annual Mattermost subscription, you agree to provide quarterly reports of the actual number of registered users within your system. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.
 
-We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. Please ensure your screenshot is taken from the top of the page and includes the **Total Active Users** metric.
+We'll send you an email notice around the end of the quarter reminding you to send us your report. To send this report, take a screenshot of **System Console > Site Statistics** and send in a reply email back to us. 
```

Comment:
Can we incorporate this illustration for this?

<img width="1112" alt="Screenshot 2023-01-25 at 2 51 15 PM" src="https://user-images.githubusercontent.com/101303489/214673979-a39553a1-73f5-4a14-a4cf-467a6c4f38de.png">

Reply:
We definitely can!